packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -74,9 +89,12 @@ interface ThumbnailProps extends GenericProps {
     theme?: Theme;
     /** Variant. */
     variant?: ThumbnailVariant;
-
     /** Focal Point coordinates. */
     focusPoint?: FocusPoint;
+    /** Allows to re-center the image according to the focal point after after window resizing */
+    updateOnWindowResize?: boolean;
```

Comment:
Maybe we can update the variable name to understand that the expected value is a boolean.
Something like: isFollowingWindowSize

Reply:
Yep, boolean values are always prefixed by is or has.